saskatoon/harvest/admin.py

```diff
@@ -66,8 +68,52 @@ class PropertyImageInline(admin.TabularInline):
 class PropertyAdmin(LeafletGeoAdmin):
     model = Property
     inlines = [PropertyImageInline]
+    list_display = (
+        '__str__',
```

Comment:
`__str__` does not include postal_code at the moment, please add the postal_code to list_display